src/pages/Login/Login.jsx

```diff
@@ -1,8 +1,190 @@
+            <form className="form">
+              <div className="form-inner">
+                <span className="login-right-font">이메일</span>
+                <input
+                  type="text"
+                  id="username"
+                  className="login-input-size"
+                  value={email}
+                  onChange={handleEmailInput}
+                ></input>
+                {email.length === 0 || isEmailValid || (
+                  <div className="email-input-demand login-right-font">
+                    이메일 형식이 아닙니다.
+                  </div>
+                )}
+                <div>
+                  <span className="login-option login-right-font">
+                    다른 로그인 옵션:
+                  </span>
+                  <a
+                    href="#"
+                    className="login-option-link link login-right-font"
+                  >
+                    일회용 코드로 로그인
+                  </a>
+                </div>
+                <div className="email-space"></div>
+                <span class="password-name login-right-font">비밀번호</span>
+                <input
+                  type="password"
+                  id="password"
+                  className="login-input-size"
+                  onChange={handlePasswordInput}
+                ></input>
+                {password.length === 0 || isPasswordValid || (
+                  <div className="email-input-demand login-right-font">
+                    비밀번호는 8자 이상, 영문자,숫자,특수문자 포함입니다.
+                  </div>
+                )}
+                <span className="login-right-font link">비밀번호 찾기</span>
+                <div className="password-space-down"></div>
+                <div className="login-status">
+                  <div className="login-status-box">
+                    <input type="checkbox" className="login-checkbox"></input>
+                    <div className="login-checkbox-label login-right-font">
+                      로그인 상태 유지
+                    </div>
+                  </div>
+                  <img
+                    src="images/alert.png"
+                    alt="alert"
+                    className="login-alert"
+                  />
+                </div>
+                <div className="login-space-up"></div>
+                <button
+                  type="button"
+                  class="login-button login-right-font"
+                  onClick={handleLogin}
+                  disabled={isEmailValid && isPasswordValid ? false : true}
+                >
+                  로그인
+                </button>
+              </div>
+            </form>
+            <div className="footer-space-up"></div>
+            <div className="login-right-footer">
+              <div className="right-footer-inner">
+                <span className="login-right-font login-footer-make">
+                  IWEA 계정이 없으신가요? 지금 바로 만들어보세요.
+                </span>
+                <div className="footer-space"></div>
```

Comment:
필요한 내용일까요?
margin 같은 걸로 해결할 수는 없을까요?

만약 필요하다고 하신다면
```suggestion
                <div className="footer-space" />
```
내용이 없는 태그는 바로 닫아주는 습관을 들여주세요!

Reply:
@CheolhoKim-1105 다른 부분에도 내용 없는 태그가 많은데 확인 후 닫아주세요!